### DISCUSSION TITLE
Realm JavaScript v10.20.0-beta.0

### DISCUSSION THREAD

**Discussion by kraenhansen** [realm/realm-js#4196] [General]
NOTE: This is an early (beta) release with Hermes/JSI support.
NOTE: This release is rebased on our `10.11.0` release and as such contain the same enhancements and fixes.
NOTE: Due to an unrelated CI issue, the prebuilt binary for Node.js on Windows is not available for this release, please install using `--ignore-scripts` to skip the step downloading or building the prebuilt binary when installing this version on Windows.

### Enhancements
* Catching missing libjsi.so when loading the librealm.so and rethrowing a more meaningful error, instructing users to upgrade their version of React Native.

### Fixed
* Fixed support of user defined classes that don't extend `Realm.Object`.
* Fixed throwing "Illegal constructor" when `new` constructing anything other than `Realm` and `Realm.Object`.

### Compatibility
* MongoDB Realm Cloud.
* Realm Studio v11.0.0.
* APIs are backwards compatible with all previous releases of Realm JavaScript in the 10.5.x series.
* File format: generates Realms with format v22 (reads and upgrades file format v5 or later for non-synced Realm, upgrades file format v10 or later for synced Realms).

<hr /><em>This discussion was created from the release <a href='https://github.com/realm/realm-js/releases/tag/v10.20.0-beta.0'>Realm JavaScript v10.20.0-beta.0</a>.</em>

**Comment by Waltari10** [comment]
So what do you think about using this version in production? I guess the Alpha version wasn't recommended, but what about this one?

**Comment by lalitkumarext** [comment]
Hii 

I'm facing a Missing  Realm constructor. "pod install" error in android and ios.
